### PR TITLE
fix!: Remove Most Fields from dbt Target

### PR DESCRIPTION
**Pull request by eakmanrq**

### CODE REVIEW

File: sqlmesh/dbt/target.py [hunk 1]
```diff
@@ -42,6 +42,14 @@
     t.Type[IncrementalUnmanagedKind],
 ]
 
+# We only serialize a subset of fields in order to avoid persisting sensitive information
+SERIALIZABLE_FIELDS = {
+    "type",
+    "name",
+    "database",
+    "schema_",
```
Comment (crericha) [inline sqlmesh/dbt/target.py:50]: Is this field serialized as `schema_` or `schema`. It needs to be the latter.

**Review by crericha — APPROVED** [review]

File: sqlmesh/dbt/target.py (hunk 1 above)
Comment (crericha) [reply sqlmesh/dbt/target.py:50]: Unit test confirms its the latter.

Comment (eakmanrq) [reply sqlmesh/dbt/target.py:50]: lol was wondering the same

**Comment by eakmanrq** [comment]
@izeigerman do you mind also taking a look when you get a chance (I know you are traveling)? Just want to get another set of eyes since this is breaking/requires a migration.

**Pull request merged**